.travis.yml

```diff
@@ -0,0 +1,6 @@
+dist: bionic
+
```

Comment:
Can you add the following somewhere in this file to reduce noise?
```
# Don't email me the results of the test runs.
notifications:
  email: false
```

Similar to this: https://github.com/CMSgov/bcda-app/blob/master/.travis.yml#L24-L26